api/src/main/java/org/eclipse/microprofile/health/spi/SPIFactory.java

```diff
@@ -19,14 +19,16 @@
  * SPDX-License-Identifier: Apache-2.0
  *
  */
-package org.eclipse.microprofile.health;
+
+package org.eclipse.microprofile.health.spi;
+
+import org.eclipse.microprofile.health.ResponseBuilder;
 
 /**
- * The basic health check procedure interface
- *
- * @author Heiko Braun
- * @since 13.06.17
+ * Created by hbraun on 07.07.17.
  */
-public interface HealthCheckProcedure {
-    HealthStatus perform();
+public interface SPIFactory {
+
+    ResponseBuilder createResponseBuilder();
+
```

Comment:
@heiko-braun This whole PR is misleading. The structure is incorrect in my view. I would rather redo this PR instead of moving to a different PR. You can close this PR but not commit it and start a new one if you like.